src/utils/constants.js

```diff
@@ -826,3 +826,5 @@ export const allTagsForOnlineClass = [
   "programmingClasses",
   "duplicate",
 ];
+
+export const INPUT_PATTERNS = { name: /^[\p{L}\p{M}']+$/u , replaceName: /[^\p{L}\p{M}']+/u}
```

Comment:
You'll want to use a `g` flag to this regex, so it makes a global replace in case the user pastes in a name. For example, if they paste the name `1a2b3c`, it'll only replace the first `1` instead of the `1`, `2`, and `3` without the `g` flag, allowing the invalid `a2b3c` name.
```suggestion
export const INPUT_PATTERNS = { name: /^[\p{L}\p{M}']+$/u , replaceName: /[^\p{L}\p{M}']+/gu}
```